src/components/account-switcher.jsx

```diff
@@ -63,11 +66,25 @@ const AccountSwitcher = () => {
       });
     }
   };
+  const resetBalance = async () => {
+    const account = JSON.parse(login_information.active_account);
+    sendRequest({
```

Comment:
Nit: Can we refactor this code with Async and await, to keep the promise handling consistant